app/webpack/reactjs/components/art_modal.tsx

```diff
@@ -27,7 +22,7 @@ interface ArtModalWindowProps {
 const ArtModalWindow: FC<ArtModalWindowProps> = ({ artPiece, handleClose }) => {
   setAppElement("body");
 
-  const { artPieces } = useContext(ArtPiecesContext);
+  const { artPieces } = useArtPiecesContext();
```

Comment:
This is the basic change.
No need for this component to know about `ArtPiecesContext` if we can isolate that in the file where it's defined.

We don't do this at work currently and we have the context and a hook and a provider all kind of defined in different places which feels a little gross.  

I think wrapping it all up in one file (see art_pieces.context.ts) is a nicer way to do it.